warehouse/accounts/forms.py

```diff
@@ -59,7 +59,7 @@ class PreventNullBytesValidator:
 
     def __init__(self, message=None):
         if message is None:
-            message = _("Null bytes are not allowed.")
+            message = _("Please enter a value.")
```

Comment:
This is a loss of information: this validator is *specifically* checking for null bytes (i.e. ASCII NULL), so "Please enter a value" doesn't describe what it's actually rejecting.

Or in other words: `\x00invalid` *is* a value, it's just not a valid one. But with this change we'd emit a confusing error message suggesting that it isn't a value.